defaults/main.yml

```diff
@@ -105,7 +105,7 @@ rke2_data_path: /var/lib/rancher/rke2
 rke2_artifact_url: https://github.com/rancher/rke2/releases/download/
 
 # Local path to store artifacts
-rke2_artifact_path: /rke2/artifact
+rke2_artifact_path: /var/tmp
```

Comment:
If we are gonna change it, let's change it to the path which is used in the RKE2 Docs https://docs.rke2.io/install/airgap (/root/rke2-artifacts)